.github/actions/release/bump-versions/action.yml

```
@@ -75,6 +59,22 @@ runs:
       run: |
         cargo set-version ${{ inputs.version }}
 
+    - name: Replace identity_iota version in Wasm bindings
+      shell: bash
+      if: ${{inputs.release-target == 'rust'}}
+      working-directory: bindings/wasm
+      run: |
+        cargo add identity_iota --path=../../identity_iota
+
+    - name: Replace identity versions in stronghold nodejs bindings
+      shell: bash
+      if: ${{inputs.release-target == 'rust'}}
+      working-directory: bindings/stronghold-nodejs
+      run: |
+        cargo add identity_core --path=../../identity_core
+        cargo add identity_iota_core_legacy --path=../../identity_iota_core_legacy
+        cargo add identity_account_storage --path=../../identity_account_storage
```

The Stronghold bindings now rely on `identity_did` too.

```suggestion
        cargo add identity_core --path=../../identity_core
        cargo add identity_did --path=../../identity_did
        cargo add identity_iota_core_legacy --path=../../identity_iota_core_legacy
        cargo add identity_account_storage --path=../../identity_account_storage
```